.github/actions/check-current-workflow-is-latest/action.yml

```diff
@@ -0,0 +1,30 @@
+name: Check current workflow is the latest workflow
+description: Checks whether the current workflow is the most recent workflow.
```

Comment:
Similar to other internally uses actions, add "Intentionally not documented in Actions.md since it's only meant for internal use."